.github/workflows/_shared-first-time-setup.yml

```
@@ -0,0 +1,64 @@
+name: "Repository First Time Setup automation (shared)"
+
+on:
+  workflow_call:
+    inputs:
+        esf_version:
+          type: string
+          default: '7.2.2'
+          required: true
+        archetype_url:
+          type: string
+          default: 'https://changeme'
+          required: true
+        groupId:
+          type: string
+          default: ''
+          required: true
+        artifactId:
+          type: string
+          default: ''
+          required: true
+
+jobs:
+  init:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Download ESF archetype
+      uses: carlosperate/download-file-action@v1
+      with:
+        file-url: ${{ github.event.inputs.archetype_url }}
+      if: startsWith(${{ github.event.inputs.archetype_url }}, 'https://eth-repo.s3.amazonaws.com/esf')
+
+    - name: Run the maven install
+      run: |
+        mvn install:install-file -Dfile=esf-addon-archetype-${{ github.event.inputs.esf_version }}.jar \
+        -DgroupId=com.eurotech.framework -DartifactId=esf-addon-archetype \
+        -Dversion=${{ github.event.inputs.esf_version }} -Dpackaging=jar
+      shell: bash
```

This fails with:

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-install-plugin:2.4:install-file (default-cli) on project standalone-pom: The specified file '/home/runner/work/release_notes_automation/release_notes_automation/esf-addon-archetype-7.2.2.jar' not exists -> [Help 1]
```

I would suggest using the file path which is the output of the download step

```suggestion
    - name: Download ESF archetype
      uses: carlosperate/download-file-action@v1
      id: download-file
      with:
        file-url: ${{ github.event.inputs.archetype_url }}
      if: startsWith(${{ github.event.inputs.archetype_url }}, 'https://eth-repo.s3.amazonaws.com/esf')

    - name: Run the maven install
      run: |
        mvn install:install-file -Dfile=${{ steps.download-file.outputs.file-path }} \
        -DgroupId=com.eurotech.framework -DartifactId=esf-addon-archetype \
        -Dversion=${{ github.event.inputs.esf_version }} -Dpackaging=jar
      shell: bash
```